### DISCUSSION TITLE
DAI 2023-2024 - Practical work 2 - Olin Bourquin and Kevin Farine

### DISCUSSION THREAD

**Discussion by OlinB** [heig-vd-dai-course/heig-vd-dai-course#154] [Show and tell]
We are doing a blackjack server and client (maybe multi-client if we have time)

**Comment by OlinB** [comment]
[Repo here](https://github.com/OlinB/heig-vd-dai-pw-2)

**Reply by KevinFarine** [reply]
Code commit : 
commit a02c17d

**Reply by OlinB** [reply]
[lien](https://github.com/OlinB/heig-vd-dai-pw-2/commit/a02c17dd09698aba4ca197369f4bf142d7ac8747)
